integration_tests/steps/device.py

```diff
@@ -75,15 +90,14 @@ def step_impl(context):
     assert context.aggregator.is_on_market_cycle_called
 
 
-@when("the external client is started with test_aggregator_ess")
+@when("the gsy-e-sdk is connecting to gsy-e with test_aggregator_ess")
 def step_impl(context):
-    # Wait for d3a to activate all areas
-    sleep(5)
-    # Connects one client to the load device
     context.aggregator = EssAggregator("storage_aggregator")
+    sleep(3)
```

Comment:
Why do we need this sleep here? Shouldn't we already be sure that we've been waiting long enough, since we're calling `wait_for_log_to_appear_in_container_logs()` in the first step?

Reply:
Yes, we do not need the sleep here, as the `is_blocking` parameter is se to True in the `RedisAggregator._connect_to_simulation`, therefore the aggregator should already be active at this point.